### PR TITLE
[NCP-3457] some hints were changed by suggestions

### PR DESCRIPTION
**Pull request by jungho73**

### CODE REVIEW

File: zh-Hans.json [hunk 1]
```diff
@@ -2046,8 +2046,8 @@
 	"reportQuery.page.title.type.reqSummary": "边缘请求数汇总",
 	"reportQuery.page.title.type.statusDetails": "状态码详情图表",
 	"reportQuery.page.title.type.volSummary": "边缘流量汇总图",
-	"reportQuery.page.validation.count.group.by": "最多只能选择两项。",
-	"reportQuery.page.validation.count.wildcard": "不按加速域名查询时，最多只能选择一个泛域名。",
+	"reportQuery.page.validation.count.group.by": "请选择最多两项。",
```
Comment (Jianghai-Quantil) [inline zh-Hans.json:2049]: Please keep the original texts.

File: zh-Hans.json [hunk 2]
```diff
@@ -2046,8 +2046,8 @@
...
+	"reportQuery.page.validation.count.wildcard": "请选择单个通配符主机名。",
```
Comment (Jianghai-Quantil) [inline zh-Hans.json:2050]: Please keep the original text.

**Review by Jianghai-Quantil — CHANGES_REQUESTED** [review]
Requested some changes.

**Review by burtonquantil — APPROVED** [review]
The English text looks OK.

**Comment by jungho73** [comment]
> Requested some changes.

kept the original texts.

**Review by Jianghai-Quantil — APPROVED** [review]
OK. Good to merge.

**Pull request merged**